packages/fieldset/test/lion-fieldset.test.js

```diff
@@ -9,19 +9,34 @@ import {
 } from '@open-wc/testing';
 import sinon from 'sinon';
 import { localizeTearDown } from '@lion/localize/test-helpers.js';
-import '@lion/input/lion-input.js';
+import { LionInput } from '@lion/input';
 
 import '../lion-fieldset.js';
 
+customElements.define(
+  'complex-input',
+  class extends LionInput {
+    parser() {
+      return this.modelValue;
+    }
+
+    formatter(mv) {
+      return mv && mv.value !== undefined ? mv.value : mv;
+    }
+  },
+);
```

Comment:
please use `defineCE` helper from open-wc to prevent name collisions in our tests

remember that we run all tests (more than 700 now) on the same page without iframes?

Reply:
Chances are slim and readability will go down, but will do ;)